packages/contentstack-utilities/src/date-time.ts

```diff
@@ -0,0 +1,15 @@
+export const formatDate = (date: Date) => {
+  return [
+    date.getFullYear(),
```

Comment:
Improve the file name form date-time to the actual purpose, like date-time-formatter

Reply:
The date-time itself is self-explanatory, implying that it can encompass various date-time-related utilities without the necessity of being strictly formatted dates or time.